js/views/device_pairing_dialog_view.js

```diff
@@ -117,10 +116,7 @@
       this.pubKey = this.pubKeyRequests.pop();
     },
     async confirmUnpairDevice() {
-      await libloki.storage.removePairingAuthorisationForSecondaryPubKey(
-        this.pubKeyToUnpair
-      );
-      await lokiFileServerAPI.updateOurDeviceMapping();
+      this.trigger('deviceUnpairingRequested', this.pubKeyToUnpair);
```

Comment:
Couldn't we just do `Whisper.events.trigger('deviceUnpairingRequested', pubKey)` directly from here?

Reply:
Yes we could, I didn't do it on purpose to decouple the view code from the business logic.
The idea was to have AppView (app_view.js) passing on events between the specific views and the rest of the code.